sys/log/full/syscfg.yml

```diff
@@ -32,6 +32,12 @@ syscfg.defs:
         description: 'Limits what level log messages are compiled in.'
         value: 0
 
+    LOG_FLAGS_LOG_IMG_HASH:
+        description: >
+            Enable logging of the first 4 bytes of the image hash. 0 - disable; 
+            1 - enable.
+        value: 0
```

Comment:
I think this should be enabled by default, also the `LOG_VERSION` should be bumped up in the syscfg.yml.

Reply:
Bumped up in LOG_VERSION = 4 and enabling image hash logging by default.